lib/ansible/module_utils/docker_common.py

```diff
@@ -457,6 +457,43 @@ def get_container(self, name=None):
 
         return result
 
+    def get_network(self, name=None):
```

Comment:
How about allowing to pass the correct (and complete) ID here as well? Then the list networks step would not be needed.